analyzers/tests/SonarAnalyzer.UnitTest/Extensions/AssignmentExpressionSyntaxExtensionsTests.cs

```diff
@@ -43,13 +393,18 @@ public class AssignmentExpressionSyntaxExtensionsTests
         [DataRow("var (_, _) = (1, 2);", "")]
         public void AssignmentExpressionSyntaxExtensions_AssignmentTargets_DeconstructTargets(string assignment, string expectedTargets)
         {
-            var syntaxTree = CSharpSyntaxTree.ParseText(WrapInMethod(assignment));
-            var assignmentExpression = syntaxTree.GetRoot().DescendantNodesAndSelf().OfType<AssignmentExpressionSyntax>().First();
-            var allTargets = assignmentExpression.AssignmentTargets();
+            var allTargets = ParseAssignmentExpression(assignment).AssignmentTargets();
```

Comment:
This test should have this signature to avoid the `string.Join` indirection on both sides
```
AssignmentTargets_DeconstructTargets(string assignment, params string[] expectedTargets)
```